lib/iris/coords.py

```diff
@@ -1578,6 +1578,28 @@ def bounds(self, bounds):
         self._bounds = bounds
 
 
+class CellMeasures(AuxCoord):
+    """
+    CF Cell Measures coordinate. Sublasses AuxCoord to provide string
+    representation for cube attributes list.
+
+    """
+
+    def __init__(self, points, cf_measures, standard_name=None, long_name=None,
+                 var_name=None, units='1', bounds=None, attributes=None,
+                 coord_system=None):
+
+        AuxCoord.__init__(self, points, standard_name=None,
+                          long_name=None, var_name=None, units='1',
+                          bounds=None, attributes=None, coord_system=None)
```

Comment:
I see now, thanks for the clarification. That was a little careless on our part!